### PR TITLE
Fix docs for JS return types in Window API 

### PR DESCRIPTION
**Pull request by jangler**
# Description

These Window API functions return promises, not raw values. This was not documented. Also fixed a typo in the relevant document.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

- [ ] Windows
- [ ] macOS
- [ ] Linux
  
## Test Configuration

n/a

# Checklist:

- [x] I have updated `website/src/pages/changelog.mdx` with details of this PR
- [x] My code follows the general coding style of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

Hopefully the PR number I predicted in the changelog edit turns out to be right.

## Summary by CodeRabbit

- **New Features**
	- Introduced the ability to exit the app via right-click on the dock icon.

- **Documentation**
	- Updated the Window API documentation to correctly reflect return types and Promise-based behavior.

### CODE REVIEW

**Comment by leaanthony** [comment]
Thanks for this! Could you possibly edit the non-versioned docs as that means we'll get to keep it as the versions are cycled. Thank you.

**Comment by jangler** [comment]
And the "edit this page" button made it sound so easy...

I also noticed that the return type syntax is inconsistent on that page (colon vs. no colon), but I'm not planning on doing anything about that.

File: website/docs/reference/runtime/window.mdx [hunk 1]
```diff
@@ -124,7 +124,7 @@ JS: `WindowSetSize(width: number, height: number)`
 Gets the width and height of the window.
 
 Go: `WindowGetSize(ctx context.Context) (width int, height int)`<br/>
-JS: `WindowGetSize() : Size`
+JS: `WindowGetSize(): Promize<Size>`
```
[the remark this answers is not in the document]
Comment (jangler) [reply website/docs/reference/runtime/window.mdx]: Thanks robot

**Review by leaanthony — APPROVED** [review]
Thanks @jangler

**Pull request merged**